.github/workflows/container-publish.yaml

```diff
@@ -0,0 +1,102 @@
+name: Container image build and publish
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '45 17 * * *'
+  push:
+    branches: [ "master" ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
This is necessary or otherwise every pull request will have a failure for the action, however, the if condition on line 27 makes it so the action is skipped: https://github.com/RedHatInsights/insights-puptoo/pull/466/files#diff-89107e2d6691e5508ba3037dc77d6cba57367296e45b24d6ca543d06116dd22aR27